protobuf/transform.go

```diff
@@ -167,6 +182,14 @@ func (t *Transformer) transformStruct(pkg *Package, s *scanner.Struct) *Message
 	return msg
 }
 
+func defaultOptionsForScannedMessage(s *scanner.Struct) Options {
+	opts := make(Options)
+
+	opts["(gogoproto.drop_type_declaration)"] = NewLiteralValue("true")
+
+	return opts
```

Comment:
this implementation seems to be the same than used in
`defaultOptionsForScannedEnum(e *scanner.Enum) Options`
Why not abstracting it, and reusing it in both places?